### PR TITLE
docs: update fabricator.rst

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- update fabricator.rst

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/testing/fabricator/005.php [hunk 1]
```diff
@@ -2,28 +2,30 @@
 
 namespace App\Models;
 
+use Generator;
+
```
Comment (paulbalandan) [inline user_guide_src/source/testing/fabricator/005.php:6]: should this be Faker's Generator?

File: user_guide_src/source/testing/fabricator/021.php [hunk 2]
```diff
@@ -2,15 +2,18 @@
 
 namespace App\Models;
 
+use CodeIgniter\Test\Fabricator;
+use Generator;
```
Comment (paulbalandan) [inline user_guide_src/source/testing/fabricator/021.php:6]: Faker\Generator ?

File: user_guide_src/source/testing/fabricator/005.php (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/testing/fabricator/005.php]: Thanks! Fixed.

File: user_guide_src/source/testing/fabricator/021.php (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/testing/fabricator/021.php]: Fixed

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**